packages/go/analysis/analysis.go

```diff
@@ -72,7 +72,7 @@ func ParseKind(rawKind string) (graph.Kind, error) {
 
 func ParseKinds(rawKinds ...string) (graph.Kinds, error) {
```

Comment:
What a strange artifact that the Kind parsing code lives in analysis. I took a quick look at where it and the underlying ParseKind are used, and it doesn't appear that this should cause any harm. It's virtually unused by analysis (ParseKind is used by `ein` and that's the closest any of this gets to being actually analysis related). I don't see a great place to put this outside of analysis, especially since some tests rely on it, so I'm more documenting this here as something weird but that is currently benign.